src/template-tags/tickets.php

```diff
@@ -14,7 +14,7 @@
 	/**
 	 * Returns the current post object that can have tickets attached to it
 	 *
-	 * Optionally the post object or ID of a ticket post can be passed in and, again, the
+	 * Optionally the post object or ID of a ticketed post can be passed in and, again, the
```

Comment:
I found this docblock a bit unclear (expected a different outcome) so I changed it to make it a bit more clear.